sessionctx/variable/varsutil.go

```diff
@@ -27,6 +28,8 @@ import (
 	"github.com/pingcap/tidb/types"
 )
 
+const secondsPerYear = 31536000
```

Comment:
Normal Year or leap year?

Reply:
MySQL don't consider leap year here. More than 31536000 seconds cause truncate warning.

Reply:
Then 365*24*60*60 maybe much easier to understand?

Reply:
@zhexuany  I've fixed here. PTAL :)